eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -159,7 +159,59 @@ function Get-java-UpdatedDocsMsToc($toc) {
     # Add services exsting in old toc but missing in automation.
     $otherService = $services[-1]
     $sortableServices = $services | Where-Object { $_ –ne $otherService }
-
+    foreach ($service in $sortableServices) {
+        if ($service.name -eq "SQL") {
+            $items = $service.items
+            $service.items = @(
+                [PSCustomObject]@{
+                    name  = "Client"
```

Comment:
This doesn't need to be `Client` it can be the `DisplayName` of the package in the metadata.